project/Build.scala

```diff
@@ -1376,7 +1376,12 @@ object Build {
         generateDocumentation(Testcases)
       }.value,
 
-      generateReferenceDocumentation := Def.taskDyn {
+      generateReferenceDocumentation := Def.inputTaskDyn {
+        val shouldRegenerateExpectedLinks = spaceDelimited("[--no-regenerate-expected-links]").parsed match {
+          case "--no-regenerate-expected-links" :: rest => false
+          case _ => true
+        }
```

Comment:
The idea is to _not_ pass the option `--no-regenerate-expected-links` in another workflow? Or is this used by some workflow already?

I suggest maybe turning things around and naming the option `--regenerate-expected-links`. Also, I suggest using the `literal` constructor:

```suggestion
        val shouldRegenerateExpectedLinks = literal("--regenerate-expected-links").?.parsed.isDefined
```

Reply:
Ah true. OK. Do you mind still following my suggestion of using `literal` instead of `spaceDelimited`?

Reply:
Of course, I'm not so familiar with sbt parsers so I need to trust you :)

Reply:
I applied the change